### PR TITLE
[NPU C++] Update model support & examples & benchmark 

### PR DESCRIPTION
**Pull request by plusbang**
## Description

- Add llama & minicpm support
- Update examples (scripts and remove outdated description)
- Update benchmark script

### 4. How to test?
- [x] Application test
(Verified Qwen2-1.5B & Llama2-7B & Llama3-8B & Llama3.2-1B & Llama3.2-3B & Minicpm-1B & Minicpm-2B CW)

### CODE REVIEW

File: python/llm/example/NPU/HF-Transformers-AutoModels/LLM/README.md [hunk 1]
```diff
@@ -85,55 +85,45 @@ done
...
-#### 32.0.100.2625
-Supported models: Llama2-7B, MiniCPM-1B, Baichuan2-7B
-#### 32.0.101.2715
-Supported models: Llama3-8B, MiniCPM-2B, Qwen2-1.5B, Qwen2.5-7B
-
 ### Run
 ```cmd
 :: to run Llama-2-7b-chat-hf
-python llama.py
+python llama2.py --save-directory <converted_model_path>
```
Comment (rnwang04) [inline python/llm/example/NPU/HF-Transformers-AutoModels/LLM/README.md:102]: --repo-id-or-model-path ?

File: python/llm/example/NPU/HF-Transformers-AutoModels/LLM/README.md [hunk 2]
```diff
@@ -85,55 +85,45 @@ done
...
 
-:: to run Meta-Llama-3-8B-Instruct (LNL driver version: 32.0.101.2715)
-python llama.py --repo-id-or-model-path meta-llama/Meta-Llama-3-8B-Instruct
+:: to run Meta-Llama-3-8B-Instruct
+python llama3.py --save-directory <converted_model_path>
```
Comment (rnwang04) [inline python/llm/example/NPU/HF-Transformers-AutoModels/LLM/README.md:105]: --repo-id-or-model-path ?

File: python/llm/example/NPU/HF-Transformers-AutoModels/LLM/README.md [hunk 3]
```diff
@@ -85,55 +85,45 @@ done
...
 
-:: to run Qwen2.5-3B-Instruct (LNL driver version: 32.0.101.2715)
-python qwen.py --repo-id-or-model-path Qwen/Qwen2.5-3B-Instruct --low_bit sym_int8
+:: to run Qwen2.5-3B-Instruct
+python qwen.py --repo-id-or-model-path Qwen/Qwen2.5-3B-Instruct --low_bit sym_int8 --save-directory <converted_model_path>
 
-:: to run Qwen2.5-7B-Instruct (LNL driver version: 32.0.101.2715)
-python qwen.py --repo-id-or-model-path Qwen/Qwen2.5-7B-Instruct
+:: to run Qwen2.5-7B-Instruct
+python qwen.py --save-directory <converted_model_path>
```
Comment (rnwang04) [inline python/llm/example/NPU/HF-Transformers-AutoModels/LLM/README.md:120]: --repo-id-or-model-path ？

File: python/llm/example/NPU/HF-Transformers-AutoModels/LLM/README.md [hunk 4]
```diff
@@ -85,55 +85,45 @@ done
...
 
 :: to run MiniCPM-1B-sft-bf16
-python minicpm.py
+python minicpm.py --save-directory <converted_model_path>
```
Comment (rnwang04) [inline python/llm/example/NPU/HF-Transformers-AutoModels/LLM/README.md:123]: --repo-id-or-model-path ?

**Review by rnwang04 — APPROVED** [review]
others LGTM

File: python/llm/example/NPU/HF-Transformers-AutoModels/LLM/README.md (hunk 1 above)
Comment (plusbang) [reply python/llm/example/NPU/HF-Transformers-AutoModels/LLM/README.md]: > --repo-id-or-model-path ?

yeah, have updated the default value in the command.

File: python/llm/example/NPU/HF-Transformers-AutoModels/LLM/qwen.py [hunk 5]
```diff
@@ -52,9 +52,6 @@
     parser.add_argument('--low_bit', type=str, default="sym_int4",
```
Comment (Oscilloscope98) [inline python/llm/example/NPU/HF-Transformers-AutoModels/LLM/qwen.py:52]: `--low_bit` -> `--low-bit` :)

**Pull request merged**